src/class-elements/field-definition-accessor-no-line-terminator.case

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022 Chengzhong Wu. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Valid accessor FieldDefinition, ClassElementName, PropertyName Syntax
+info: |
+  FieldDefinition[Yield, Await] :
+    accessor [no LineTerminator here] ClassElementName[?Yield, ?Await] Initializer[+In, ?Yield, ?Await]opt
+
+template: default
+features: [decorators]
+---*/
+
+//- elements
+accessor
+$;
+static accessor
+$;
+
+//- assertions
+let c = new C();
+
+assert.sameValue(Object.hasOwnProperty.call(C.prototype, 'accessor'), false);
+assert.sameValue(Object.hasOwnProperty.call(C.prototype, '$'), false);
+assert.sameValue(Object.hasOwnProperty.call(C, 'accessor'), true);
+assert.sameValue(Object.hasOwnProperty.call(C, '$'), true);
+assert.sameValue(Object.hasOwnProperty.call(c, 'accessor'), true);
+assert.sameValue(Object.hasOwnProperty.call(c, '$'), true);
```

Comment:
🥳

Reply:
I see that this `Object.hasOwnProperty.call` exists elsewhere, but I think that was a mistake and the intention was actually to use `Object.prototype.hasOwnProperty.call`. I'm not going to block on it here, but someone should do a cleanup to use `Object.prototype.hasOwnProperty.call` or `Reflect.hasOwnProperty`.

Reply:
I can do it in a follow-up PR if you are ok with it.